00XX-deploy-eu868-region-plan-to-the-majority-of-africa

```diff
@@ -0,0 +1,321 @@
+# **HIP XX: Deploy EU868 region plan to the majority of Africa**
+
+
+
+* Author: Adrian Clint, [https://github.com/waveform06](https://github.com/waveform06) and Helium Foundation [https://github.com/dewi-alliance](https://github.com/dewi-alliance) 
+* Start Date: 22 Sep 2003
+* Category: Technical
+* Original HIP PR:
+* Tracking Issue:
+* Vote Requirements: veIOT
+
+## 
+**Summary**
+
+
+The LoRaWAN Alliance has a country by country bottom up approach to allocation of frequencies and does not push for harmonization of a frequency to a region as a top down approach.
+
+The Helium Foundation recommends that the community provisionally sets the currently unspecified and EU433 only countries in the Africa region to the EU868 frequency plan to match with what we understand from local regulatory authorities like the Africa Telecoms Union (ATU) and local deployers have defined as the future LoRaWAN regional frequency EU868. The LoRa Alliance does not currently specify regional parameters for these countries, or only has EU433 as an available frequency plan, so this would be an addition that the Helium Foundation is proposing for Helium network specific devices.
```

Comment:
```suggestion
The Helium Foundation recommends that the community provisionally sets the currently unspecified and EU433 only countries in the Africa region to the EU868 frequency plan to match with what we understand from local regulatory authorities like the Africa Telecoms Union (ATU) and local deployers have defined as the future LoRaWAN regional frequency EU868. The LoRa Alliance does not currently specify regional parameters for these countries, or only has EU433 as an available frequency plan, so this would be an addition that the Helium Foundation is proposing for Helium network specific devices. The LoRa Alliance has a country-by-country, or bottom up, approach to allocation of frequencies.
```